### ISSUE TITLE
New Chunks Not Generating.

### ISSUE THREAD

**Issue by tealtwo** [mircokroon/minecraft-world-downloader#589]
New Chunks Not Generating.

Copy and save world, put files in another sever/world, try to load new chunks/

 - Game: Minecraft 1.20
 - Downloader: latest

**Comment by StrafeYourself** [comment]
Yeah I know man it is pretty annoying but mircokroon is a bloody legend and if there is anyone that can get it done it is him. However, remember it is good to donate as he is a developer and they are in high demand. So if we pay him money for getting these done with this project he can develop it a lot more faster as he is then making income from doing this project.

**Comment by StrafeYourself** [comment]
However, most of us man are just kids bro so the fact that this is free in the first place is good enough as it is. As he is giving us a chance to make amazing thumbnails for yt videos witch eventually make money so the right thing to do with that money it is put some money towards the thing that got you that money as a way of saying thank you.
